UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FulfilmentDataService.cs

```diff
@@ -211,5 +176,104 @@ private void CreateIsoAndSha1ForExchangeSet(List<FssBatchFile> fileDetails, stri
                 }
             });
         }
+
+        private async Task<string> GetProductDataSinceDateTimeFromEss(string sinceDateTime)
+        {
+            ExchangeSetResponseModel exchangeSetResponseModel = await _essService.GetProductDataSinceDateTime(sinceDateTime);
+            if (!string.IsNullOrEmpty(exchangeSetResponseModel.Links.ExchangeSetBatchDetailsUri.Href))
+            {
+                string essBatchId = CommonHelper.ExtractBatchId(exchangeSetResponseModel.Links.ExchangeSetBatchDetailsUri.Href);
+                _logger.LogInformation(EventIds.BatchCreatedInESS.ToEventId(), "Batch is created by ESS successfully with BatchID - {BatchID} | {DateTime} | _X-Correlation-ID : {CorrelationId}", essBatchId, DateTime.Now.ToUniversalTime(), CommonHelper.CorrelationID);
+
+                return essBatchId;
+            }
+            else
```

Comment:
Don't need else here as the if contains a return.

Reply:
Removed